hooks/useConversation.ts

```diff
@@ -51,24 +52,53 @@ const useConversation = (
       }
       setLoading(false)
     }
+    listMessages()
+  }, [conversation, convoMessages])
+
+  useEffect(() => {
+    if (!conversation) return
     const streamMessages = async () => {
-      const stream = await conversation.streamMessages()
-      setStream(stream)
+      stream = await conversation.streamMessages()
       for await (const msg of stream) {
         if (dispatchMessages) {
           await dispatchMessages({
             peerAddress: conversation.peerAddress,
             messages: [msg],
           })
         }
+        if (
+          latestMsgId !== msg.id &&
+          Notification.permission === 'granted' &&
+          msg.senderAddress !== walletAddress &&
+          !browserVisible
+        ) {
+          new Notification('New Message On XMTP', {
+            body: `From ${msg.senderAddress}`,
+            icon: '/xmtp-icon.png',
```

Comment:
Is there a way to prevent the icon from being displayed when expanding the notification?

<img width="364" alt="CleanShot 2022-09-16 at 21 43 08@2x" src="https://user-images.githubusercontent.com/510695/190840822-ad72c2ee-fbb7-4b3f-bfe6-3b81295fc802.png">
<img width="369" alt="CleanShot 2022-09-16 at 21 43 13@2x" src="https://user-images.githubusercontent.com/510695/190840823-af927e04-d3df-4bab-a798-86b3c740e99a.png">

Reply:
@darickdang Can I request some feedback around this, like what should be the design?

Reply:
Design request filed: https://github.com/xmtp-labs/hq/issues/682